frontend/src/components/BottomNav.js

```diff
@@ -35,6 +36,13 @@ export default function BottomNavi() {
       showLabels
       className={classes.root}
     >
+      {jwtDecoded && (
+        <BottomNavigationAction
+          value={"/home"}
+          label="Home"
+          icon={<HomeIcon />}
+        />
+      )}
```

Comment:
```suggestion
      {jwtDecoded && (<>
        <BottomNavigationAction
          value={"/home"}
          label="Home"
          icon={<HomeIcon />}
        />
        {/* other actions go here*/}
      </>)}
```
Use a fragment to group single items and make react happy